```
+ nrnivmodl mod
/jenkins/10/workspace/hpc.module-testing/ringtest-parallel
Mod files: "mod/halfgap.mod"

Creating x86_64 directory for .o files.

COBJS=''
 -> [32mCompiling[0m mod_func.c
 -> [32mNMODL[0m ../mod/halfgap.mod
Translating halfgap.mod into /jenkins/10/workspace/hpc.module-testing/ringtest-parallel/x86_64/halfgap.c
Thread Safe
 -> [32mCompiling[0m halfgap.c
 => [32mLINKING[0m shared library ./libnrnmech.so
ld: cannot find -limf
ld: cannot find -lsvml
ld: cannot find -lirng
ld: cannot find -lipgo
ld: cannot find -ldecimal
ld: cannot find -lcilkrts
make: *** [mech_lib_shared] Error 1
```

Looks like there is an intel issue somehow. By comparing with a successful run, we now do: 
```
 ++ export INTEL_ROOT
```

--- a/deploy/environments/compilers.yaml
+++ b/deploy/environments/compilers.yaml
@@ -4,6 +4,8 @@ spack:
   modules:
     tcl:
       all:
+        environment:
+          set:: {}
         filter:
           environment_blacklist: ['CPATH', 'LIBRARY_PATH']
       whitelist:

